gin/routes.go

```diff
@@ -117,12 +117,14 @@ func InitRoutes() *gin.Engine {
 		apiadmin.GET("/destination", handleAdminGetDestinations)
 		apiadmin.GET("/view-redeemable-codes", handleAdminViewAllRedeemableCodes)
 		apiadmin.POST("/generate-five-codes", handleAdminGenerateFiveCodes)
+		apiadmin.GET("/view-users", handleAdminViewUsers)
 	}
 
 	// TODO: make a group for this and look at authentication
 	router.GET("/socket/", gin.WrapH(socket.GetServer()))
 	router.POST("/socket/", gin.WrapH(socket.GetServer()))
 
+	router.GET("/socket/", gin.WrapH(socket.Server))
```

Comment:
tại sao lại thêm dòng này?

code is not compiling

Reply:
Em xoa roi day